### PR TITLE
LW-975 Update Okta library to latest version.

### PR DESCRIPTION
**Pull request by mlkirkpatrick**

### CODE REVIEW

**Comment by rdoughty** [comment]
@mlkirkpatrick fix travis and package.json conflict

**Comment by mlkirkpatrick** [comment]
@rdoughty Fixed Travis. I'll fix the conflict when merging (don't want to merge prep into feature branch.)

**Review by rdoughty — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@ndlib/markdown-to-jsx": "^6.11.2",
-    "@okta/okta-auth-js": "^2.10.1",
+    "@okta/okta-auth-js": "^4.0.0",
```
Comment (danthewolfe) [inline package.json:13]: There's a version 4 now? 
```
(╯°□°)╯︵ ┻━┻
```

**Review by danthewolfe — APPROVED** [review]
I'm assuming you've tested that it actually works when you try to login and don't already have a session (incognito window)?

**Comment by mlkirkpatrick** [comment]
@danthewolfe yes

**Pull request merged**